src/plugins/logger.js

```diff
@@ -5,6 +5,7 @@ import { deepCopy } from '../util'
 export default function createLogger ({
   collapsed = true,
   transformer = state => state,
+  ignored = type => false,
```

Comment:
Yeah, I think passing in the entire mutation object leaves room for more flexibility.